ecosystem/sep-0030.md

```diff
@@ -140,58 +140,93 @@ A set of common fields are used to describe an account in requests and responses
+        { "type": "email", "value": "person1@example.com" }
+      ]
     }
-  }
+  ]
 }
 ```
 
-##### Example (Form)
+###### Form
 
 ```
-identities.other.account=GDUA...&identities.other.email=person1%40example.com&identities.other.phone_number=%2B10000000001&identities.owner.account=GBEA...&identities.owner.email=person2%40example.com&identities.owner.phone_number=%2B10000000002
+identities.0.auth_methods.account=GDUA...&identities.0.auth_methods.email=person1%40example.com&identities.0.auth_methods.phone_number=%2B10000000001
+```
+
+##### Example (With Role)
+
+###### JSON
+
+```json
+{
+  "identities": [
+    {
+      "role": "sender",
+      "auth_methods": [
+        { "type": "account", "value": "GDUA..." },
+        { "type": "phone_number", "value": "+10000000001" },
+        { "type": "email", "value": "person1@example.com" }
+      ]
+    },
+    {
+      "role": "receiver",
+      "auth_methods": [
+        { "type": "account", "value": "GBEA..." },
+        { "type": "phone_number", "value": "+10000000002" },
+        { "type": "email", "value": "person2@example.com" }
+      ]
+    },
+  ]
+}
+```
+
+###### Form
+
+```
+identities.0.role=sender&identities.0.auth_methods.account=GDUA...&identities.0.auth_methods.email=person1%40example.com&identities.0.auth_methods.phone_number=%2B10000000001&identities.1.role=receiver&identities.1.auth_methods.account=GBEA...&identities.1.auth_methods.email=person2%40example.com&identities.1.auth_methods.phone_number=%2B10000000002
 ```
 
 #### Common Response Fields
 
 Name | Type | Description
 -----|------|------------
 `address` | string | The Stellar account ID or address of the account.
-`identities.has_owner` | boolean | True if the account has alternative identities set for the owner identity.
-`identities.has_other` | boolean | True if the account has alternative identities set for the other identity.
+`identities` | array | A list of identities that if any one is authenticated will be able to gain control of this account.
+`identities[].role` | string | Optional, same as request. See examples above.
+`identities[].authenticated` | boolean | Optional. Present and true if the currently authenticated client is authenticated with the identity.
 `signer` | string | The signer public key that the server will sign transactions with for this account when requested by an authenticated user.
-`identity` | string | The identity of the authenticated client in relation to the account:<ul><li>`account`: Authenticated as the account with [SEP-10].</li><li>`owner`: Authenticated with an owner identity.</li><li>`other`: Authenticated with an other identity.</li></ul>
 
 ##### Example (JSON)
 
 ```json
 {
   "address": "GFDD...",
-  "identities": {
-    "has_owner": true,
-    "has_other": true
-  },
-  "identity": "account",
+  "identities": [
+    {
+      "role": "sender",
+      "authenticated": true
+    },
+    {
+      "role": "receiver"
```

Comment:
If I understood correctly, there's also a chance that you'd be authenticated for an identity that does not have a `role`, right? Should we also illustrate that here?

I'd also favor always returning all the possible fields, which means adding an `"authenticated": false` in this object.

Reply:
Correct to the first.

I can have authenticated: false returned in the other cases too 👍 .